plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -213,27 +205,39 @@ class ServiceDestroyModal extends React.Component {
         rightButtonCallback={this.handleRightButtonClick}
         showHeader={true}
       >
-        <p>
-          This action
-          {" "}
-          <strong>CANNOT</strong> be undone. This will permanently delete the
-          {" "}
-          <strong>{serviceName}</strong>
-          {" "}
-          {serviceLabel.toLowerCase()}.
-          Type ("
-          <strong>{serviceName}</strong>
-          ") below to confirm you want to delete the
-          {" "}
-          {serviceLabel.toLowerCase()}.
-        </p>
-        <input
-          className="form-control filter-input-text"
-          onChange={this.handleChangeInputFieldDestroy}
-          type="text"
-          value={this.state.serviceNameConfirmationValue}
-          autoFocus
-        />
+        {this.getGroupHeader()}
+        {(!this.isGroupWithServices() ||
+          this.state.forceDeleteGroupWithServices) &&
+          <div className="modal-service-delete-center">
```

Comment:
As discussed I was confused to see the markup part of the condition test, even though works @bstavroulakis agreed to change to not cause confusion.

Reply:
@weblancaster Addressed issues, that you for reviewing this